ros2msg/test/test_cli.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import subprocess
+
+
+def test_cli():
+    packages_cmd = ['ros2', 'msg', 'packages']
+    packages_result = subprocess.run(packages_cmd, stdout=subprocess.PIPE, check=True)
+    package_names = packages_result.stdout.decode().splitlines()
+
+    # explicit dependencies of this package will for sure be available
+    assert 'std_msgs' in package_names
+
+    count = 0
+    for package_name in package_names:
+        package_cmd = ['ros2', 'msg', 'package', package_name]
```

Comment:
The same applies to `ros2 srv package[s]` verbs.

@ros2/team can you guys weigh in on this? Do you prefer `ros2 msg package <pkg>` or `ros2 msg list --package <pkg>` (also `ros2 msg packages` vs. `ros2 msg list --packages`)?

If the consensus is that the alternative verb names I proposed would be better, then I'll open a pr to update the commands later.